ruby/ql/lib/codeql/ruby/security/CleartextLoggingQuery.qll

```diff
@@ -0,0 +1,38 @@
+/**
+ * Provides a taint-tracking configuration for "Clear-text logging of sensitive information".
+ *
+ * Note, for performance reasons: only import this file if
+ * `CleartextLogging::Configuration` is needed, otherwise
+ * `CleartextLoggingCustomizations` should be imported instead.
+ */
+
+private import ruby
+private import codeql.ruby.DataFlow
+private import codeql.ruby.TaintTracking
+import CleartextLoggingCustomizations::CleartextLogging
+private import CleartextLoggingCustomizations::CleartextLogging as CleartextLogging
+
+/**
+ * A taint-tracking configuration for detecting "Clear-text logging of sensitive information".
+ */
+class Configuration extends TaintTracking::Configuration {
+  Configuration() { this = "CleartextLogging" }
+
+  override predicate isSource(DataFlow::Node source) { source instanceof CleartextLogging::Source }
+
+  override predicate isSink(DataFlow::Node sink) { sink instanceof CleartextLogging::Sink }
+
+  override predicate isSanitizer(DataFlow::Node node) {
+    super.isSanitizer(node)
```

Comment:
`TaintTracking::Configuration.isSanitizer(_)` is `none()`, so isn't this line redundant?

Reply:
I added it in case we extend the default `isSanitizer` predicate in the future, though I haven't been consistent in doing this with all dataflow/taint tracking queries.